socotratools/socotratools/client.py

```diff
@@ -522,47 +533,54 @@ def get_endorsements_from_policy(self, policy_locator):
         return self.__get("/policies/" + policy_locator +
                           "/endorsements")
 
-    def create_renewal(self, policy_locator, end_timestamp=None,
-                       field_values={}, add_field_groups=[],
-                       update_field_groups=[], remove_field_groups=[],
-                       add_exposures=[], update_exposures=[],
-                       end_exposures=[], action=None):
-
-        data = {'renewalEndTimestamp': end_timestamp,
+    def create_renewal(self, policy_locator,
+                       field_values=None,
+                       add_field_groups=None,
+                       remove_field_groups=None,
+                       update_field_groups=None,
+                       add_exposures=None,
+                       end_exposures=None,
+                       update_exposures=None,
+                       # above: optional
+                       end_timestamp=None,
+                       state='application'):
+        data = {
                 'fieldValues': field_values,
                 'addFieldGroups': add_field_groups,
                 'updateFieldGroups': update_field_groups,
                 'removeFieldGroups': remove_field_groups,
                 'addExposures': add_exposures,
                 'updateExposures': update_exposures,
                 'endExposures': end_exposures,
-                'action': action
-                }
-
+                'renewalEndTimestamp': end_timestamp,
+                'state': state}
```

Comment:
I'm not sure why this is `state` rather than `action`, but that's what the implementation say. Rule of thumb: PATCH means `action`, whereas POST means `state`.